setup.cfg

```diff
@@ -0,0 +1,107 @@
+[metadata]
+name = sqlfluff
+description = The SQL Linter for Humans
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/sqlfluff/sqlfluff
+author = Alan Cruickshank
+author_email = alan@designingoverload.com
+license = MIT License
+license_files = LICENSE.md
+project_urls =
+    Homepage = https://www.sqlfluff.com
+    Documentation = https://docs.sqlfluff.com
+    Changes = https://github.com/sqlfluff/sqlfluff/blob/main/CHANGELOG.md
+    Source = https://github.com/sqlfluff/sqlfluff
+    Issue Tracker = https://github.com/sqlfluff/sqlfluff/issues
+    Twitter = https://twitter.com/SQLFluff
+    Chat = https://github.com/sqlfluff/sqlfluff#sqlfluff-on-slack
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: Unix
+    Operating System :: POSIX
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Utilities
+    Topic :: Software Development :: Quality Assurance
+keywords =
+    sqlfluff
+    sql
+    linter
+    formatter
+    bigquery
+    exasol
+    hive
+    mysql
+    postgres
+    redshift
+    snowflake
+    spark3
+    sqlite
+    teradata
+    tsql
+    dbt
+
+[options]
+package_dir =
+    =src
```

Comment:
Is it correct, this setting beginning with an `=`? (Looks odd.)

Reply:
@barrywhart it is correct, it's the equivalent of package data setting in previous setup.py 👍 
![image](https://user-images.githubusercontent.com/80432516/142954991-0c85132c-eaa6-48e9-a3b2-b6e9454c8807.png)

I've tested building and it works as expected 

see ref here:
![image](https://user-images.githubusercontent.com/80432516/142955070-f826fe21-87dd-41e0-b368-0a2e28682b0a.png)